test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -75,6 +75,15 @@ public EndToEndTests(ServerFixture serverFixture, ITestOutputHelper output) : ba
             }
         }
 
+        [Fact]
+        public async Task CanStartConnectionUsingDefaultTransport()
+        {
+            var url = _serverFixture.BaseUrl + "/echo";
+            var connection = new HttpConnection(new Uri(url));
```

Comment:
Add a `loggerFactory` to the `HttpConnection`, also this test passes on my home machine